README.md

```diff
@@ -15,68 +19,38 @@ conda env create -f environment.yml
 source activate minutes
 ```
 
-### :rocket: Running the Server
-
-To run the server in the docker container, run:
-
-```
-make build-prod
-make run
-```
-
-To run it using conda, run:
-
-```
-source activate minutes
-python app/main.py
-```
-
-### :rainbow: Running the Research Environment
+## Testing
 
-```
-make build-dev
-make dev
+```bash
+pytest --cov=minutes -vvv test
 ```
 
-## :point_up: Deployment
+## Building the Conda Package
 
-We deploy continuously using Travis CI and a Docker Hub deploy bot.
+Specify a new git version tag, edit the `meta.yml` and run:
 
-You can also deploy the production and development images manually if you like.
-
-```
-make push-dev
-make push-prod
+```bash
+conda-build .
 ```
 
-You will need to set `DOCKER_USERNAME` and `DOCKER_PASSWORD` and be a member of the `ubclaunchpad` docker hub organization to deploy manually.
+## Example Usage
```

Comment:
It might also be useful to provide an example of a call to `Minutes.models` that returns a list of available models, to showcase what base models come prepackaged with the library.